django/contrib/postgres/fields/serial.py

```diff
@@ -0,0 +1,61 @@
+from django.db import connection
+from django.db.models import Field
+from django.utils.translation import ugettext_lazy as _
+
+
+class SerialField(Field):
+    description = _("Serial")
+
+    empty_strings_allowed = False
+    default_error_messages = {
+        'invalid': _("'%(value)s' value must be an integer."),
+    }
+
+    def __init__(self, *args, **kwargs):
+        kwargs['blank'] = True
+        kwargs.setdefault('unique', True)
+        super(SerialField, self).__init__(*args, **kwargs)
+
+    def deconstruct(self):
+        name, path, args, kwargs = super().deconstruct()
+        kwargs.pop('blank')
+        return name, path, args, kwargs
```

Comment:
You might want to handle the `unique=True` case here also. Additional tests for deconstruction wouldn't hurt.

Reply:
The way your deconstruct is currently defined has the following behavior

```python
SerialField().deconstruct()[4] == {'unique': True, 'editable': False}
```

Because `Field.deconstruct()` considers `unique` and `editable` default values to be respectively `False`, `True`.

You'll want to use the same mechanism you use to remove `blank` here but only conditionally 

```python
name, path, args, kwargs = super().deconstruct()
kwargs.pop('blank')
if kwargs['unique']:
    kwargs.pop('unique')
if not kwargs['editable']:
    kwargs.pop('editable')
return name, path, args, kwargs
```

Notice that's the behaviour of other builtin fields that override `Field.__init__` defaults

e.g. `FilePathField`.

https://github.com/django/django/blob/e671337e8b35427b379303cbf6808855f04083e8/django/db/models/fields/__init__.py#L1667

https://github.com/django/django/blob/e671337e8b35427b379303cbf6808855f04083e8/django/db/models/fields/__init__.py#L1699-L1700

Tests should be added for `deconstruct` as well.